docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -275,7 +275,7 @@ $ rhoas context use --name development-context
 $ cd ~/app-services-guides/code-examples/quarkus-service-registry-quickstart/
 ----
 
-. Generate an environment variables file that contains the credentials for service account.
+. Create a service account and save the credentials in an environment variables file.
```

Comment:
Suggest:

"Create a service account for the Quarkus application to use. Save the credentials in an environment variables file in the directory for the producer component."